tests/rt.conf

```diff
@@ -2,270 +2,271 @@
-RUN     | fv3_ccpp_wrtGauss_netcdf                                                                                                       | standard    |                | fv3         |
-RUN     | fv3_ccpp_wrtGauss_netcdf_parallel                                                                                              | standard    |                | fv3         |
-RUN     | fv3_ccpp_wrtGlatlon_netcdf                                                                                                     | standard    |                | fv3         |
-RUN     | fv3_ccpp_wrtGauss_nemsio                                                                                                       | standard    |                | fv3         |
-RUN     | fv3_ccpp_wrtGauss_nemsio_c192                                                                                                  | standard    |                | fv3         |
-RUN     | fv3_ccpp_stochy                                                                                                                | standard    |                | fv3         |
-RUN     | fv3_ccpp_iau                                                                                                                   | standard    | wcoss_cray     | fv3         |
-RUN     | fv3_ccpp_iau                                                                                                                   | standard    | wcoss_dell_p3  | fv3         |
-RUN     | fv3_ccpp_iau                                                                                                                   | standard    | hera.intel     | fv3         |
-RUN     | fv3_ccpp_iau                                                                                                                   | standard    | orion.intel    | fv3         |
-RUN     | fv3_ccpp_iau                                                                                                                   | standard    | cheyenne.intel | fv3         |
-RUN     | fv3_ccpp_ca                                                                                                                    | standard    | hera.intel     | fv3         |
-RUN     | fv3_ccpp_ca                                                                                                                    | standard    | orion.intel    | fv3         |
-RUN     | fv3_ccpp_ca                                                                                                                    | standard    | cheyenne.intel | fv3         |
-RUN     | fv3_ccpp_lndp                                                                                                                  | standard    | hera.intel     | fv3         |
+RUN     | fv3_ccpp_control                                                                                                        | standard    |                | fv3         |
```

Comment:
I'd suggest to remove all the ccpp in the test names

Reply:
Can do, but this means several changes in `fv3_conf` files, too (linking restart files from a previous coldstart run etc). 

Originally we wanted to make this commit as small as possible and it is growing and growing. There is a lot more cleanup work to be done (see also the different steps identified in my issue). I suggest to add this as another cleanup step to the issue instead of commingling everything in one commit.